Dockerfile

```diff
@@ -102,7 +102,6 @@ ADD backend/scripts/import.sh /usr/local/bin/import
 RUN chmod +x /entrypoint.sh /usr/local/bin/backup /usr/local/bin/restore /usr/local/bin/import
 
 COPY --from=build-backend /build/backend/remark42 /srv/remark42
-COPY --from=build-backend /build/backend/templates /srv
 COPY --from=build-frontend /srv/frontend/apps/remark42/public/ /srv/web/
```

Comment:
I don't recall why we needed templates copied in the final container, but I think this was not due to statik?

Reply:
It's not necessary, and I explained how to replace any template file with your own as clearly as I could in #1450.